pkg/operator/ceph/pool/controller_test.go

```diff
@@ -348,19 +348,27 @@ func TestCephBlockPoolController(t *testing.T) {
 	pool.Spec.Mirroring.Mode = "image"
 	err = r.client.Update(context.TODO(), pool)
 	assert.NoError(t, err)
-	res, err = r.Reconcile(req)
-	assert.NoError(t, err)
-	assert.False(t, res.Requeue)
-	err = r.client.Get(context.TODO(), req.NamespacedName, pool)
-	assert.NoError(t, err)
-	assert.Equal(t, cephv1.ConditionReady, pool.Status.Phase)
+	for i := 0; i < 5; i++ {
+		res, err = r.Reconcile(req)
+		assert.NoError(t, err)
+		assert.False(t, res.Requeue)
+		err = r.client.Get(context.TODO(), req.NamespacedName, pool)
+		assert.NoError(t, err)
+		assert.Equal(t, cephv1.ConditionReady, pool.Status.Phase)
+		if _, ok := pool.Status.Info[RBDMirrorBootstrapPeerSecretName]; ok {
+			break
+		}
+		logger.Infof("FIX: trying again to update the mirroring status")
```

Comment:
There is still an issue that needs investigating with the mirroring status not being updated on the pool CR, the intention of this PR is to at least make the test more stable until we find the root cause.

Reply:
Looks good to make CI pass. I'm looking into this and trying to find the root cause.